extension/setup.js

```diff
@@ -5,6 +5,8 @@
 var setupSurvey = {};  // Namespace variable
 setupSurvey.status = constants.SETUP_PENDING;
 setupSurvey.questions = [];
+setupSurvey.completionStatus = false;  // Whether the user submitted the form.
```

Comment:
completionStatus sounds like an enum, how about renaming to isCompleted or something like that?